### DISCUSSION TITLE
Doesn't work on reference.

### DISCUSSION THREAD

**Discussion by yoghurtdewdew** [YU000jp/logseq-plugin-confirmation-done-task#21] [Q&A]
I will use block reference for todos to keep clean of checkbox (leave with only one checkbox). I found it doesn‘t work on block reference as it didn’t show the confirmation option when I click on the checkbox of block reference.

**Comment by YU000jp** [comment]
I will include it in the release. In the case of a reference, if changes are made to the original block, I will detect it and assign the DONE property. If you want to display it, you need to either make the block a task without referencing it or embed it using 'embed'.
